tsup.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
```

Comment:
Also extracted config from package.json and specify entry points. With upcoming updates we can changed imports to root package for easier moving files around later